client/src/components/AppModal.jsx

```diff
@@ -124,14 +124,14 @@ class AppModal extends React.Component {
               </X>
             </CloseButton>
           </Container>
-          <StyledRatingOverview average={this.state.ratings.average} numReviews={this.state.reviews.length} isModal={true}/>
-          <StyledRatingGraphs ratings={this.state.ratings} isModal={true}/>
+          <StyledRatingOverview average={this.state.ratings.average} numReviews={this.state.reviews.length} isModal={true} />
```

Comment:
Style guide will ask you to always  de-structure these.

Reply:
Will do!